### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found some misspelling with `typos`.
I've left "invokable" alone.

### CODE REVIEW

File: test/core/010.phpt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 --TEST--
-ReflectionMethod::__toString() tests (overriden method)
```
Comment (Ocramius) [inline test/core/010.phpt]: These tests are imported from PHP-SRC, and should not be touched

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: test/core/010.phpt (hunk 1 above)
Comment (szepeviktor) [reply test/core/010.phpt]: All right!
Reverting ...

Comment (szepeviktor) [reply test/core/010.phpt]: Done 🍏

**Review by Ocramius — APPROVED** [review]
Thanks @szepeviktor!

Note: due to the various revert commits, I'm squash-merging this.

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**